vividus-plugin-mobitru/src/main/java/org/vividus/mobitru/selenium/MobitruUrlPropertiesValidator.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2019-2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.mobitru.selenium;
+
+import java.net.URI;
+import java.util.Map;
+
+import org.vividus.selenium.PropertyValidator;
+import org.vividus.util.property.IPropertyParser;
+
+public class MobitruUrlPropertiesValidator extends PropertyValidator
+{
+    private static final String INCORRECT_PROPERTY_FOR_MOBITRU_MESSAGE =
+            "Incorrect 'selenium.grid.%s' property value for Mobitru: '%s'";
+    private static final String URL = "url";
+    private static final String HOST = "host";
+
+    public MobitruUrlPropertiesValidator(IPropertyParser propertyParser)
+    {
+        super(propertyParser, "selenium.grid.");
+    }
+
+    @Override
+    public void validateFilteredProperties(Map<String, String> properties)
+    {
+        String host = properties.get(HOST);
+        if (!host.contains("mobitru"))
+        {
+            throw new IllegalStateException(String.format(INCORRECT_PROPERTY_FOR_MOBITRU_MESSAGE, HOST, host));
+        }
+        String urlString = properties.get(URL);
+        String authorityFromUrl = URI.create(urlString).getAuthority();
+        String authority = properties.get("username") + ":" + properties.get("password") + "@" + host;
```

Comment:
no need to recheck credentials, it's enough to check:
- URL host is equal to host property value
- host property value ends with 'mobitru.com'